src/scripts/webpack/config.js

```diff
@@ -48,19 +48,22 @@ const getConfigType = (configFolderPath) => {
 };
 
 const nodeEnv = process.env.NODE_ENV || 'development';
-const envWhitelist = ['apiEndpoint', 'fetchLocalesFrom'];
```

Comment:
As discussed offline env variables needed by the script should be prefixed with `CONFIG_` to avoid override by node process.